modelling/src/test/java/org/axonframework/modelling/command/inspection/AnnotatedAggregateMetaModelFactoryTest.java

```diff
@@ -247,6 +267,26 @@ void testFindIdentifier() {
         assertEquals("id", inspector.routingKey());
     }
 
+    @Test
+    void testFindGetterIdentifier() {
+        AggregateModel<SomeGetterIdAnnotatedHandlers> inspector =
+                AnnotatedAggregateMetaModelFactory.inspectAggregate(SomeGetterIdAnnotatedHandlers.class);
+
+        assertEquals("SomeGetterIdAnnotatedHandlers", inspector.type());
+        assertEquals("id", inspector.getIdentifier(new SomeGetterIdAnnotatedHandlers()));
+        assertEquals("getId", inspector.routingKey());
```

Comment:
When matching the models' routing key with the routing key in the `@CommandHandler`, it is expected to be identical to the property name in the command. So, not the method name, as is being validated here. As such, we should remove any `get` prepending the routing key if it comes from a method.